packers/msx/pack_msx_game.py

```
@@ -0,0 +1,521 @@
+            print(f"❌ Invalid preset: '{preset}'")
+            print(f"   Valid presets: {', '.join(VALID_PRESETS)}")
+            sys.exit(1)
+
+    # ── Derive title ──
+    if not title:
+        title = derive_title(game_path)
+
+    # ── Find WebMSX engine ──
+    engine_file = find_engine(engine_path)
+    if not engine_file:
+        print(f"❌ WebMSX engine (wmsx.js) not found!")
```

Ruff (0.15.4)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)